### ISSUE TITLE
Glfw Error 65550: Failed to detect any supported platform 

### ISSUE THREAD

**Issue by suveerudayashankara** [nvidia-holoscan/holohub#626]
[info] [fragment.cpp:588] Loading extensions from configs...
[info] [gxf_executor.cpp:262] Creating context
[info] [gxf_executor.cpp:1764] creating input IOSpec named 'receivers:0'
[info] [gxf_executor.cpp:1764] creating input IOSpec named 'render_buffer_input'
[info] [gxf_executor.cpp:1764] creating input IOSpec named 'camera_up_input'
[info] [gxf_executor.cpp:1764] creating input IOSpec named 'camera_look_at_input'
[info] [gxf_executor.cpp:1764] creating input IOSpec named 'camera_eye_input'
[info] [gxf_executor.cpp:1764] creating input IOSpec named 'input_specs'
[info] [gxf_executor.cpp:1764] creating input IOSpec named 'receivers'
[info] [gxf_executor.cpp:2171] Activating Graph...
[info] [gxf_executor.cpp:2201] Running Graph...
[info] [gxf_executor.cpp:2203] Waiting for completion...
[info] [greedy_scheduler.cpp:191] Scheduling 2 entities
Authorization required, but no authorization protocol specified
Glfw Error 65550: Failed to detect any supported platform
[error] [gxf_wrapper.cpp:59] Exception occurred when starting operator: 'visualizer' - Failed to initialize glfw
[warning] [entity_executor.cpp:539] Failed to start entity [visualizer]
[warning] [greedy_scheduler.cpp:243] Error while executing entity 12 named 'visualizer': GXF_FAILURE
[error] [entity_executor.cpp:630] Entity [visualizer] must be in Started, Tick Pending, Ticking or Idle stage before stopping. Current state is StartPending
[info] [greedy_scheduler.cpp:401] Scheduler finished.
[error] [program.cpp:580] wait failed. Deactivating...
[error] [runtime.cpp:1649] Graph wait failed with error: GXF_FAILURE
[warning] [gxf_executor.cpp:2204] GXF call GxfGraphWait(context) in line 2204 of file /workspace/holoscan-sdk/src/core/executors/gxf/gxf_executor.cpp failed with 'GXF_FAILURE' (1)
[info] [gxf_executor.cpp:2214] Graph execution finished.
[error] [gxf_executor.cpp:2222] Graph execution error: GXF_FAILURE
Traceback (most recent call last):
File "/opt/nvidia/holoscan/examples/v4l2_camera/python/v4l2_camera.py", line 95, in
main(config_file=config_file)
File "/opt/nvidia/holoscan/examples/v4l2_camera/python/v4l2_camera.py", line 89, in main
app.run()
RuntimeError: Failed to initialize glfw
[info] [gxf_executor.cpp:291] Destroying context
![WhatsApp Image 2024-12-10 at 11 15 56 AM](https://github.com/user-attachments/assets/ed72c652-e390-4962-ad36-dc8aaa65d157)

USING rtx 4090 for the work
but i am facing problems related to GXF and video replayer related program.

but Holoscan is running well

**Comment by AndreasHeumann** [comment]
Hi,
this line of the log is indicating that there is a problem with X11 authorization and therefore the window can not be opened:
`Authorization required, but no authorization protocol specified`
If you using ssh and the `./run launch` command please add the `--ssh-x11` option to the command. This makes sure that the docker user has the rights to launch X11 apps.
Please let me know if this solved your issue.
Regards,
Andreas

**Comment by tbirdso** [comment]
Hi @suveerudayashankara , thanks for the new details. It seems that the issue you are encountering in HoloHub's High Speed Endoscopy app is likely a separate pathing issue equivalent to the one reported in #606. Could you please attempt the resolution in that [comment](https://github.com/nvidia-holoscan/holohub/issues/606#issuecomment-2514678127) and report the outcome in that issue?

Regarding the original `v4l2_camera` error you reported, could you please run the following with @AndreasHeumann 's suggestion and let us know if you continue to see the GLFW error? Note that the V4L2 application by default requires that both a V4L2 device and a display are available to the machine.
```sh
# Run the HoloHub container
cd holohub
./dev_container build
./dev_container launch --ssh_x11

# Inside the container
cd /opt/nvidia/holoscan/examples
/opt/nvidia/holoscan/examples/v4l2_camera/python/v4l2_camera.py
```

**Comment by suveerudayashankara** [comment]
yep i tried it now
@AndreasHeumann  and @tbirdso 
the holoscan is working fine
will update you with the endoscopy part 
so for now the problem is solved
thank you very much.
![Selection_002](https://github.com/user-attachments/assets/ad0fbf9a-4b35-4cc5-9c8b-06513862c4d1)

**Comment by tbirdso** [comment]
Great! I'll close this issue as resolved, please update #606 with any further discussion on the High Speed Endoscopy app.
